lib/phoenix_live_view.ex

```diff
@@ -1327,15 +1330,27 @@ defmodule Phoenix.LiveView do
 
         {:noreply, socket}
       end
+
+      def render(assigns) do
```

Comment:
`~H` is missing?

Reply:
Good catch! I'll fix this in my next pr if it's not fixed by then